### ISSUE TITLE
LoO taking way too long on average to repop

### ISSUE THREAD

**Issue by roriffxi** [EdenServer/community#232]
https://ffxiclopedia.fandom.com/wiki/Lord_of_Onzozo
https://ffxiclopedia.fandom.com/wiki/Talk:Lord_of_Onzozo

"I have camped Lord Of Onzozo for 3 months trying to get my club. In that time he has mainly poped between the 18-24hr marks. There for awhile it was usaly 18-20 the occasional end of window pop here an there. toward the end of this 3 month camping spree he started poping at end of window an even not poping tell 26 hr mark 2 days in a row then at 24 then 27hrs then server maint so pattern was reset >.< hope that helps alittle." 

I can provide more examples from other sources, but he should generally pop within 6-8 hours or so of his window opening up (his window is 16hrs on Eden afaik).  While sometimes he should be able to go beyond the 24 hour mark, he regularly goes past it.

Here are my list of kills and approximate amount of time it took between repops since i started on Eden.  I'll edit and update as time goes on.  Unless I was given a specific ToD my camping time started at 9pm PST and he would have died a minimum of 16 hours prior to me beginning the camp for the week.
All times are PST:
Oct. 8 pop approximate 11:40 am, he was "open" when I started the camp.  So at minimum this was 30 hours 40 minutes after his death (i do not have the information on his previous death).
Oct 9: 2:21 pm.  ~27 hours after death
Oct 12 8:30pm ~ 78 hours after death
Oct 13 5:40 pm ~ 21 hours after death

Oct 23 12:00 PM ~ ToD given to me was Approx 8pm on the 21s.  Approx 40 hours
Oct 24 9:30 am ~22 hrs hours after death
Oct 25 never popped.
Oct 26 never popped.
Oct 26 5pm ~ 81.5 hours after previous death

On average over 38.4+ hrs per repop is what I've been experiencing on Eden.

If anyone who has also camped here would like to add to it, I'd appreciate the input.

I can also provide approx times for 19 kills on Kupo, but I imagine that isn't as useful.

**Comment by roriffxi** [comment]
I'll start tracking total number of kills. I will say with confidence that I kill at a minimum 30 phs a day on days where he doesn't spawn.

**Comment by roriffxi** [comment]
22 kills today, no pop (then handed over to Buddies, hopefully they track kills).

**Comment by godmode** [comment]
Have broadened the window quite a bit. Please continue tracking PH kills and encourage Buddies members to do the same. Can continue posting results on this ticket.

**Comment by roriffxi** [comment]
Popped at 11:11pm PST 11/4 after 9 PHs.  Talking to Loki, he had been at the camp since 10:30am.

So it took roughly 35-40 PHs to get it to pop.

**Comment by godmode** [comment]
I know Buddies aren’t doing a great job at keeping kill counts for it but if you are able to continue counting kills I would appreciate it. Estimating how many kills are done between time points is pretty finicky

**Comment by roriffxi** [comment]
Todays pop took 33 PHs.

**Comment by roriffxi** [comment]
It was reported that today took 2 phs.

**Comment by roriffxi** [comment]
Since my last post LoO has popped at 6, 2, 4, 30+ (no pop before turn over).  Then 50+ again the next week.   (26 hours before turned to me, then it took me 33 more after that to get it to pop).  
Next day was 8.
Currently on 52 PHs and counting.

I think the logic behind (25 on average) is flawed / too low.  When looking at the sources posted above, 25 PHs would be approximately 8 hours of camping.  Assuming 16 hours for the window to open and most reports having him popped by the 18 to 24 hour mark, then 25 PHs on average would mean only about half of the time would he be popped by the 24 hour mark, rather than him popping by that mark most of the time.

**Comment by roriffxi** [comment]
@godmode Were you assuming a 5min repop on PH, instead of 18min, when the current spawn chance was added?

**Comment by godmode** [comment]
I believe I was using 16m when doing the math

**Comment by roriffxi** [comment]
https://www.bluegartr.com/threads/41937-Kraken-Club-Update/page4
"Considering b4 the droprate was at like 2-3KC/month it should drop once within 100 BCs when you do a vague math."

"People are saying they get a Kraken once every two weeks from LoO, thats pretty good. So.. 1 KC per 14 kills? "

Page 5:
"I went 0/29, then 1/30, 2/31 and 3/89. No drop in the next 54 kills(consecutive)"

"2/5 here on LoO."

"0/20 loo, missed 1 day of camping for a late night ulli camp-- RMT got a kraken drop-___-"

Page 6:

"So.. 30 kills in a month of LoO gets you a kraken club before, 30000 kills in a month of the BC gets you a kraken club now."

Page 15
"LoO over a similar time period don't you? Wasn't it like 1-2 a week?"

"More like 10 per year..."

"Considering I was about 0/62 on LoO with Kraken, for a three month time period.. I'd say... no it wasn't 1-2 per week."

"its more than 10 per year, I camped it for almost 2 months strait, managed 3 clubs. Overall about 10 dropped. And I remember one guy who camped for 6 months strait everyday...never got a club, but alot of others did..he said it averaged to about a 1/10"

"They definitely did nerf duping at some point in time, so that's a legit reason to believe that there are a lot of clubs, but I think it's more due to the fact that the drop rate, while bad, isn't TERRIBLE."

Page 16
"the drop rate off LoO averages out to be around 1 club every 3 weeks from my experience.

this is based on a pretty large sample sale, the group i camped with literally killed every LoO pop for around a year-18months and I think by the time we stopped we were around 24 clubs."
------------------------------------------------------------------------------------------------
I'm sure theres more information in the thread but I'm seeing a lot of evidence of about once every 2-3 weeks,(1 in 14-21 kills) once a month at the low end.  Also I would bet all my gil that the vast-vast-vast-vast majority of these kills were done with TH2.

On Eden there have been approximately 14 weeks since TH was turned back on and only 2 drops.  With all those kills being done with TH3.
